tests/launcher.py

```diff
@@ -40,7 +60,7 @@ class nodeDefinition:
     base_p2p_port: ClassVar[int] = 9876
     base_http_port: ClassVar[int] = 8888
     #file_size: int
-    instance_name: str = field(init=False, repr=False)
+    #instance_name: str = field(init=False, repr=False)
```

Comment:
Is there a future plan to reintroduce this, or should it just be deleted?

Reply:
Deleted this and numerous other bits of commented code and cruft.